ztag/schema.py

```diff
@@ -106,6 +106,31 @@ class CensysString(WhitespaceAnalyzedString):
     "timestamp":DateTime(),
 })
 
+ztag_dsa_params = SubRecord({
+    "p":IndexedBinary(),
+    "q":IndexedBinary(),
+    "g":IndexedBinary(),
+    "y":IndexedBinary(),
+})
+
+ztag_ecdsa_public_key = SubRecord({
+    "pub":IndexedBinary(),
+    "b":IndexedBinary(),
+    "gx":IndexedBinary(),
+    "gy":IndexedBinary(),
+    "n":IndexedBinary(),
+    "p":IndexedBinary(),
+    "x":IndexedBinary(),
+    "y":IndexedBinary(),
+    "curve":Enum(),
```

Comment:
If this is an enum, you should define the values. Otherwise it should just be a String.